java/de/yessoft/android/mohammad/ui/MainActivity/fragments/MySpecialORM.java

```diff
@@ -0,0 +1,42 @@
+package de.yessoft.android.mohammad.ui.MainActivity.fragments;
+
+import android.util.Log;
+
+import com.google.firebase.firestore.CollectionReference;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.QuerySnapshot;
+
+import java.util.List;
+
+import de.yessoft.android.mohammad.model.NewsItem;
+import de.yessoft.android.mohammad.ui.MainActivity.fragments.NewsFeedRV.NewsFeedAdapter;
+
+class MySpecialORM {
```

Comment:
bad name,

In software development , choosing good name is very important ,
and it is not easy task

Reply:
Yep